packages/formatter-html/src/assets/images/scan/sub-section.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32.6 31.8"><style>.a{fill:#4046dd}</style><path d="M11.3 14.9L5 8.6c-.5-.5-.5-1.4 0-1.9l2.1-2.1C7.6 4 8.5 4 9 4.6l6.3 6.3c.4.4.5 1 .2 1.5l1.6 1.6c1.1-1.4 1-3.5-.3-4.8L10.6 3C9.2 1.6 6.9 1.6 5.5 3L3.4 5.1C2 6.5 2 8.8 3.4 10.2l6.3 6.3c1.3 1.3 3.3 1.4 4.8.3l-1.6-1.6c-.6.2-1.2.1-1.6-.3zm6.8.1l1.6 1.6c.5-.3 1.1-.2 1.5.2l6.3 6.3c.5.5.5 1.4 0 1.9l-2.1 2.1c-.5.5-1.4.5-1.9 0l-6.3-6.3c-.4-.4-.5-1-.2-1.5l-1.6-1.6c-1.1 1.4-1 3.5.3 4.8l6.3 6.3c1.4 1.4 3.7 1.4 5.1 0l2.1-2.1c1.4-1.4 1.4-3.7 0-5.1l-6.3-6.3c-1.3-1.3-3.4-1.4-4.8-.3z" class="a"/><path d="M22.2 21.8c-.5.5-1.4.5-2 0L10.4 12c-.5-.5-.5-1.4 0-2 .5-.5 1.4-.5 2 0l9.9 9.9c.5.5.5 1.4-.1 1.9z" class="a"/></svg>
```

Comment:
This can be smaller:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32.6 31.8"><path fill="#4046dd" d="M11.3 14.9L5 8.6c-.5-.5-.5-1.4 0-1.9l2.1-2.1C7.6 4 8.5 4 9 4.6l6.3 6.3c.4.4.5 1 .2 1.5l1.6 1.6c1.1-1.4 1-3.5-.3-4.8L10.6 3a3.6 3.6 0 0 0-5.1 0L3.4 5.1a3.6 3.6 0 0 0 0 5.1l6.3 6.3a3.7 3.7 0 0 0 4.8.3l-1.6-1.6c-.6.2-1.2.1-1.6-.3zm6.8.1l1.6 1.6c.5-.3 1.1-.2 1.5.2l6.3 6.3c.5.5.5 1.4 0 1.9l-2.1 2.1c-.5.5-1.4.5-1.9 0l-6.3-6.3c-.4-.4-.5-1-.2-1.5l-1.6-1.6c-1.1 1.4-1 3.5.3 4.8l6.3 6.3a3.6 3.6 0 0 0 5.1 0l2.1-2.1a3.6 3.6 0 0 0 0-5.1l-6.3-6.3a3.7 3.7 0 0 0-4.8-.3z"/><path fill="#4046dd" d="M22.2 21.8c-.5.5-1.4.5-2 0L10.4 12c-.5-.5-.5-1.4 0-2 .5-.5 1.4-.5 2 0l9.9 9.9c.5.5.5 1.4-.1 1.9z"/></svg>
```